tools/coredumper/README

```diff
@@ -0,0 +1,28 @@
+Coredump-reader
```

Comment:
why do we have the same info in two different locations? It's a recipe for misaligned instructions

Reply:
deleted